.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  pull_request:
+  push:
+    branches: main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        stack-yaml:
+          - stack.yaml
+          - stack-lts-17.1.yaml
+          - stack-lts-15.3.yaml
```

Comment:
Are you planning to do this cross-resolver testing?

If so, you need to make these files and pass `matrix.stack-yaml` to the action at L29.

I don't usually start out with multi-GHC testing, personally, so the other option is the [Single GHC box here](https://renaissancelearning.atlassian.net/wiki/spaces/EN/pages/1442513923/Open+Source+Maintenance#Migrate-to-GitHub-Actions).

Reply:
Moved to single, thanks!